src/Components/ADT3DSceneBuilder/Internal/VisualRuleForm/Internal/ConditionsList.tsx

```diff
@@ -106,30 +176,41 @@ const ConditionsList: React.FC<IConditionsListProps> = (props) => {
     // Update list everytime valueRanges and expressionType change
     useEffect(() => {
         setConditions(getConditionItems(valueRanges, expressionType));
-    }, [valueRanges, expressionType]);
-
-    // Callbacks
-    const handleOpenFlyout = useCallback(() => {
-        // TODO: Open callout
-        alert('New condition');
-    }, []);
+    }, [valueRanges, expressionType, getConditionItems]);
 
     return (
-        <div className={classNames.container}>
-            <Stack>
-                <CardboardList<Condition>
-                    listKey={LIST_KEY}
-                    items={conditions}
+        <>
+            <div className={classNames.container}>
+                <Stack>
+                    <CardboardList<Condition>
+                        listProps={{
+                            id: LIST_KEY
+                        }}
+                        listKey={LIST_KEY}
+                        items={conditions}
+                    />
+                    <ActionButton
+                        id={'visual-rule-add-condition'}
+                        data-testid={'visual-rule-add-condition'}
+                        styles={classNames.subComponentStyles.addButton?.()}
+                        onClick={handleOpenNewConditionFlyout}
+                    >
+                        {t('3dSceneBuilder.visualRuleForm.newCondition')}
+                    </ActionButton>
+                </Stack>
+            </div>
+            {calloutInfo.isOpen && (
```

Comment:
do we need to add/remove from teh DOM here? doesn't the callout internally have show/hide logic based on isOpen? (also can probably move that to the calloutType)